.github/workflows/scripts/eslint_disable_check.py

```
@@ -0,0 +1,159 @@
+        re.IGNORECASE | re.DOTALL,
+    )
+
+    try:
+        with open(file_path, "r", encoding="utf-8") as file:
+            content = file.read()
+            return bool(eslint_disable_pattern.search(content))
+    except FileNotFoundError:
+        print(f"File not found: {file_path}")
+    except PermissionError:
+        print(f"Permission denied: {file_path}")
+    except (IOError, OSError) as e:
```

Ruff (0.14.10)

Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)